airbyte-integrations/connectors/source-mixpanel/source_mixpanel/source.py

```diff
@@ -100,12 +111,22 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         logger.info(f"Using start_date: {config['start_date']}, end_date: {config['end_date']}")
 
         auth = self.get_authenticator(config)
-        streams = [
+        streams = []
+        for stream in [
```

Comment:
we should also try try reading from the [streams with dynamically generated schema](https://github.com/airbytehq/airbyte/pull/27252/files#diff-ec2bcf88395f3b6aa5b5016d6d83b6b0c0270640e6cbb699c04034b9c9c4e0d4R133-R135) because they do not get filtered out with the current solution.